### PR TITLE
Remove all Monkey Gamemode

### LINKED ISSUE #7031: Remove monkey mode from code
Remove the monkey gamemode from the code. The gamemode is no longer maintained or supported.

### PR DESCRIPTION
**Pull request by Miliviu**
## About The Pull Request

closes #7031 
It's my first contribution to a SS13 Codebase so I would suggest a quite deep check and test from a Maintainer even if it was just removing code

## Why It's Good For The Game

Removes a completely broken game mode that didn't had no maintenance at all, in my opinion Monkey can be a fun mode, but it needs someone to mantain and remaster it.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

image
(Monkey is not available to choose as game mode)

image
(Code compiles)

</details>

## Changelog
:cl:
del: Removed Monkey Antagonist, gamemode and Disease
/:cl:

### CODE REVIEW

**Review by PowerfulBacon — DISMISSED** [review]
Hi there, thanks for the contributions.

Where the PR body says "address [issue]", it should use either "closes", "resolves" or "fixes" as these the keywords that github accepts.
Please also provide a screenshot in the testing section showing that the code compiles and the monkey game-mode is in-fact no longer in the selectable gamemodes. (use the game panel verb, then select gamemode and see if monkey is no longer present)

**Comment by Miliviu** [comment]
> Hi there, thanks for the contributions.
> 
> Where the PR body says "address [issue]", it should use either "closes", "resolves" or "fixes" as these the keywords that github accepts. Please also provide a screenshot in the testing section showing that the code compiles and the monkey game-mode is in-fact no longer in the selectable gamemodes. (use the game panel verb, then select gamemode and see if monkey is no longer present)

Right, thank you! 
It is done!

File: code/modules/mob/living/simple_animal/hostile/gorilla/gorilla.dm [hunk 1]
```diff
@@ -71,10 +71,6 @@
 		if(prob(80))
 			var/atom/throw_target = get_edge_target_turf(L, dir)
 			L.throw_at(throw_target, rand(1,2), 7, src)
-			if(diseased)
```
Comment (PowerfulBacon) [inline code/modules/mob/living/simple_animal/hostile/gorilla/gorilla.dm:74]: Can the diseased variable be removed, or is it used elsewhere?

File: config/game_options.txt [hunk 2]
```diff
@@ -116,7 +116,6 @@ PROBABILITY GANG 0
 PROBABILITY HERESY 0
 PROBABILITY INTERNAL_AFFAIRS 0
 PROBABILITY METEOR 0
-PROBABILITY MONKEY 0
```
Comment (PowerfulBacon) [inline config/game_options.txt:119]: #CONTINUOUS MONKEY on line 145 needs removing

**Review by PowerfulBacon — COMMENTED** [review]
Seems fine other than these

**Comment by AnCopper** [comment]
Merging from Master, smh.

File: code/modules/mob/living/simple_animal/hostile/gorilla/gorilla.dm (hunk 1 above)
Comment (Miliviu) [reply code/modules/mob/living/simple_animal/hostile/gorilla/gorilla.dm:74]: Yeah, you are totally right, removed!

File: code/datums/saymode.dm [hunk 3]
```diff
@@ -62,25 +62,3 @@
 		AI.holopad_talk(message, language)
 		return FALSE
 	return TRUE
-
-/datum/saymode/monkey
-	key = "k"
-	mode = MODE_MONKEY
```
Comment (Archanial) [inline code/datums/saymode.dm:68]: Was this define removed?

Comment (Miliviu) [reply code/datums/saymode.dm:68]: It was not, tysm!

**Review by PowerfulBacon — APPROVED** [review]

**Review by Crossedfall — APPROVED** [review]
Config applied

**Pull request merged**